docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -287,49 +290,52 @@ as it is now and a suggestion for new language.
 
 If you want to add a new language, then follow the translation steps above with
 some modifications. Before you add a language contact the Zonemaster project
-to discuss timeplan and other aspects of the new language. Every language must
-be update each time a message is added or changed.
+to discuss timeplan and other aspects of the new language. Every language should
+be updated at every new release of Zonemaster if there are updated or added
+messages to be translated.
 
 * First, as in the translation steps, create the branch to work in.
   ```
   git checkout origin/develop
   git checkout -b translation-update
   ```
 
-The new language is not there and cannot be updated. Instead you have to
-create the new language file (*PO* file). The easiest way is to make a copy
-of an existing file.
+There is no PO file for new language, and it has to be created. The easiest way
+is to make a copy of an existing file.
+
+[ISO 639-1]: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
+[Wikipedia]: https://www.wikipedia.org/
 
-* Determine what the language code of the new language should be. I must be
-  a code that is available in the *locale* system. Try the following commands
-  to see if it is available. Replace "xx" with that code that you think it
-  should be.
+* Determine what language code to use for the new language should be. It should
```

Comment:
Maybe remove `for the new language should be`?

Reply:
Fixed.